ddtrace/contrib/tornado/decorators.py

```diff
@@ -16,6 +16,7 @@ def _finish_span(future):
     span = getattr(future, FUTURE_SPAN_KEY, None)
 
     if span:
+        # DEV: exc_info no longer supported in future for tornado with python>=3.7
```

Comment:
I always wondered why some comments started with `DEV`. I mean, do we expect anyone else than a developer to read this? 🤔